app/test/unit/server/index.js

```diff
@@ -16,8 +16,8 @@ module.exports = function client() {
         var
           body = res.text,
 
-          contains = function (body) {
-            return JSON.stringify(body).indexOf('Hello, world!') !== -1;
+          contains = function (text) {
```

Comment:
Rename to `response` or `res`? It's not actually text until it gets stringified. =)